inscriptions/ISic2932.xml

```diff
@@ -178,22 +183,17 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="PHI" n="PHI331425">
+            <div type="edition" xml:space="preserve" xml:lang="grc" resp="#VM"><!-- this slab is re-used later for a latin inscription. Do we split the doc in two? -->
```

Comment:
I am very confused about this one; SEG thinks that it is CIG 4.9533, but Ferrua doesn't (and I have that one separately recorded as ISic1009). Until I have sight of Ferrua 1946-47 (will get it from the library next week), I'm not sure I can resolve this; the Latin text should certainly be given a separate number, but I'll deal with that at the same time as resolving the Greek text in full... leave as is for now I think....